agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannel.java

```diff
@@ -282,6 +300,11 @@ private CompletableResultCode internalSend(
     return result;
   }
 
+  // not including 401/403/503 in this list because those are commonly returned by proxy servers
+  // when they are not configured to allow traffic for westus-0
+  private static final Set<Integer> RESPONSE_CODES_INDICATING_REACHED_BREEZE =
+      new HashSet<>(asList(200, 206, 402, 408, 429, 439, 500));
```

Comment:
307/308 are in the spec.  The spec hasn't been approved/merged.  We need to be in sync.

Reply:
see discussion at https://github.com/microsoft/ApplicationInsights-Java/pull/2221#discussion_r848964564

Reply:
I will add comment in the code also to explain why they are missing

Reply:
comment added

Reply:
that makes sense for reaching redirect up to 10 times.. but what about 1 time 307/308 that is successful, should we consider that as one time success from breeze?

Reply:
as we discussed offline, I think we ignore the intermediate 307/308s and only factor in the "final" response code (after any redirects)

Reply:
https://github.com/microsoft/Telemetry-Collection-Spec/pull/58/commits/3eb967e786a310c9c34bbb06cac0ab75f59cef50

I updated the spec to reflect this.